app/validation/validate_schema_text.py

```diff
@@ -0,0 +1,101 @@
+import re
+from jsonpointer import resolve_pointer
+
+
+def get_pointer_keys():
```

Comment:
I'm not sure duplicating some of the methods of the translations repo is the best approach here.

Given these are likely to change over time - it probably makes sense that eq-translations is installed as dependency here and then the relevant utility methods within it are used to locate pointers of interest. Much like our runner does to translate schemas at the moment.

Reply:
Have refactored and added eq-translations as a dependency